backend/globaleaks/models/config.py

```diff
@@ -161,6 +160,12 @@ def initialize_config(session, tid, mode):
         for name in inherit_from_root_tenant:
             variables[name] = root_tenant_node[name]
 
+    variables['max_msg_external_to_whistle'] = 1
```

Comment:
If the variables and their defualt value is configured correctly this code is not necessary.